lua/lint/linters/trivy.lua

```diff
@@ -8,24 +9,59 @@ return {
   cmd = "trivy",
   stdin = false,
   append_fname = true,
+  -- FIXME: '--scanners config' is deprecated in favor of '--scanners misconfig'
+  -- NOTE: excluding '-q/--quiet' flag to receive log on stderr before the JSON on stdout.
+  --       The log may contain errors and warnings that can be used to notify the user this script
+  --       needs maintenance.
   args = { "--scanners", "config", "--format", "json", "fs" },
   stream = "stdout",
   ignore_exitcode = false,
-  parser = function(output, bufnr)
+  parser = function(
+    output, -- output from trivy (a JSON string)
+    bufnr -- nvim buffer number. Used to choose the correct Result from the JSON output
+  )
     local diagnostics = {}
+
+    -- TODO: Parse log messages from trivy that appear before the JSON.  Register any ERRORS or
+    --       WARNINGS as diagnostics so someone will know to make future updates to this script.
+    -- NOTE: Don't know how to get access to both stderr and stdout, so unable to address this
```

Comment:
HELP: What is mechanism to report errors/warnings during linter run?  That handled elsewhere?